### ISSUE TITLE
city with Locale.FRANCE does not use seed

### ISSUE THREAD

**Issue by PascalSchumacher** [DiUS/java-faker#65]
With 0.8 this always fails:

```java
@Test
public void testCityWithLocaleFranceAndSeed() {
    long seed = 1L;
    Faker firstFaker = new Faker(Locale.FRANCE, new Random(seed));
    Faker secondFaker = new Faker(Locale.FRANCE, new Random(seed));
    assertEquals(firstFaker.address().city(), secondFaker.address().city());
}
```

It worked with 0.7.

`city` works with a seed and without a locale.  `city` also works with a seed and with `Locale.GERMANY`.

Other methods seem to work.

**Comment by codingricky** [comment]
Fixed with this 66aa3c6c7816e07af825a3e6a7896e64605ddaa3. 

Thanks for raising it.

**Comment by PascalSchumacher** [comment]
Thanks for fixing. :)
